.github/workflows/emscripten.yml

```diff
@@ -24,7 +24,9 @@ jobs:
       uses: seanmiddleditch/gha-setup-ninja@master
 
     - name: Download and Configure Emscripten SDK
-      uses: mymindstorm/setup-emsdk@v9
+      uses: mymindstorm/setup-emsdk@master
+      with: 
+        version: 2.0.30
```

Comment:
Can we define this version so that it is shared between action files? We will have to add action files for 4 other submodules, it will be inconvenient to change version in 15 places